pkg/scripts/configs.go

```diff
@@ -40,6 +40,19 @@ if [[ -f "{{ .WORK_DIR }}/cfg/podnodeselector.yaml" ]]; then
 	sudo chown root:root /etc/kubernetes/admission/podnodeselector.yaml
 	sudo chown root:root /etc/kubernetes/admission/admission-config.yaml
 fi
+`
+
+	encryptionProvidersConfigTemplate = `
+if [[ -f "{{ .WORK_DIR }}/cfg/{{ .FILE_NAME }}" ]]; then
+	sudo mkdir -p /etc/kubernetes/encryption-providers/
+	sudo mv {{ .WORK_DIR }}/cfg/{{ .FILE_NAME }} /etc/kubernetes/encryption-providers/{{ .FILE_NAME }}
+	chmod 600 /etc/kubernetes/encryption-providers/{{ .FILE_NAME }}
+	sudo chown root:root /etc/kubernetes/encryption-providers/{{ .FILE_NAME }}
+fi
+`
+
+	deleteEncryptionProvidersConfigTemplate = `
+	sudo rm -rf /etc/kubernetes/encryption-providers/{{ .FILE_NAME }}
```

Comment:
```suggestion
	sudo rm -rf /etc/kubernetes/encryption-providers/*
```

I suggest to consider what will happen with old file if we change to other filename? Probably it make sense to cleanup all files and then render the correct one.